<ui version="4.0" >
 <class>CustomJobDialog</class>
 <widget class="QDialog" name="CustomJobDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Custom Job</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Job Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="jobNameLineEdit" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Command:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="commandLineEdit" />
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="commandPushButton" >
     <property name="text" >
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Arguments:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="argumentsLineEdit" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Working directory:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="workingDirLineEdit" />
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="workingDirPushButton" >
     <property name="text" >
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="4" column="1" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string/>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="currentModelRadioButton" >
        <property name="text" >
         <string>Current</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="fileModelRadioButton" >
        <property name="text" >
         <string>File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="modelLineEdit" />
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="modelPushButton" >
        <property name="text" >
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Data:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QLineEdit" name="dataLineEdit" />
   </item>
   <item row="8" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QPushButton" name="dataPushButton" >
     <property name="text" >
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="4" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Model (PDB):</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>jobNameLineEdit</tabstop>
  <tabstop>commandLineEdit</tabstop>
  <tabstop>commandPushButton</tabstop>
  <tabstop>argumentsLineEdit</tabstop>
  <tabstop>workingDirLineEdit</tabstop>
  <tabstop>workingDirPushButton</tabstop>
  <tabstop>currentModelRadioButton</tabstop>
  <tabstop>fileModelRadioButton</tabstop>
  <tabstop>modelLineEdit</tabstop>
  <tabstop>modelPushButton</tabstop>
  <tabstop>dataLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomJobDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomJobDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
